sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -154,6 +154,9 @@ object ScalaJSPlugin extends AutoPlugin {
     val emitSourceMaps = SettingKey[Boolean]("emitSourceMaps",
         "Whether package and optimize stages should emit source maps at all", BPlusSetting)
 
+    val scalaJSOutputWrapper = TaskKey[(String, String)]("scalaJSOutputWrapper",
+      "Custom wrapper for the generated .js files", BPlusTask)
```

Comment:
I think it would be helpful to add a note that this is a header and a footer. Wrapper might not be clear enough.